.travis.yml

```diff
@@ -1,7 +1,14 @@
 language: node_js
 node_js:
-- '0.11'
-- '0.10'
+- '4'
+env:
+  - CXX=g++-4.8
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.8
```

Comment:
@adborden (these are the versions of node)[https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Provided-Node.js-Versions] that travis supports.  After (v.4 Node requires a compiler)[https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Node.js-v4-(or-io.js-v3)-compiler-requirements].  Since it's in a docker container anyways, is the extra compiler code a big deal?